docs/upgrade.md

```diff
@@ -0,0 +1,20 @@
+# Upgrade
```

Comment:
I think it's more natural to nest this under the Installation section.

Reply:
@ammario are you referring to the `install.md` page or the /install directory?

Reply:
I'm referring to the directory.